osl/interface_linux.go

```diff
@@ -241,8 +241,9 @@ func (n *networkNamespace) AddInterface(srcName, dstPrefix string, options ...If
 	if n.isDefault {
 		i.dstName = i.srcName
 	} else {
-		i.dstName = fmt.Sprintf("%s%d", i.dstName, n.nextIfIndex)
-		n.nextIfIndex++
+		var dstPrefix = i.dstName
```

Comment:
You do not need to declare the `dstPrefix` variable. Just reuse the `dstPrefix` from the function input parameters.